src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  text-transform: uppercase;
+}
+
+.header {
```

Comment:
More correctly to add side padding for header instead of adding margin right|left for logo/navigation
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/93f66384-9870-4878-9526-0043666589c4)